src/ophys_etl/decrosstalk/decrosstalk_schema.py

```diff
@@ -0,0 +1,121 @@
+import marshmallow.exceptions
+import argschema
+
+
+class RoiSchema(argschema.ArgSchema):
+    id = argschema.fields.Int(description='ID', required=True)
+    x = argschema.fields.Int(description='x origin', required=True)
+    y = argschema.fields.Int(description='y origin', required=True)
+    width = argschema.fields.Int(description='width', required=True)
+    height = argschema.fields.Int(description='height', required=True)
+    valid_roi = argschema.fields.Bool(description='validity', required=True)
+    mask_matrix = argschema.fields.List(
+                        argschema.fields.List(argschema.fields.Bool),
+                        description='mask',
+                        cli_as_single_argument=True)
+
+
+class PlaneSchema(argschema.ArgSchema):
+
+    ophys_experiment_id = argschema.fields.Int(
+                             description='experiment ID',
+                             required=True)
+
+    motion_corrected_stack = argschema.fields.Str(
```

Comment:
I've updated the field to be an `InputFile`.

Because the name of the field involves changes to the LIMS strategy that generates the input.json, I'm going to leave it as `motion_corrected_stack` (I think this is the formal name of that field in the LIMS database)

Reply:
Kat wrote a special `H5InputFile` field. I think it was because `InputFile` was not being properly validated on windows (at that time, Isaak was working in windows.) It doesn't affect production, but, it is here:
https://github.com/AllenInstitute/ophys_etl_pipelines/blob/94d8610013dae5f1a4e67249b5654bd64c13308b/src/ophys_etl/schemas/fields.py#L5-L20